templates/support/index.html

```diff
@@ -274,13 +277,15 @@ <h2>Apps support</h2>
         <tr>
           <th>&nbsp;</th>
           <th>&nbsp;</th>
-          <th class="u-align--center"colspan="2"><h4>Ubuntu advantage</h4></th>
+          <th class="u-align--center"colspan="2"><h4>Ubuntu Pro</h4></th>
```

Comment:
The copy doc makes it look like this should be deleted. If it should stay, it should be:

`<h3 class="p-heading--4">Ubuntu Pro</h3>`

And I don't think it should be contained within a table row

Reply:
Deleted.

Reply:
Waiting to get a confirmation if this is to be changed from a table or not..

Reply:
Table format is fine. But had to add ROS too.